### PR TITLE
Update Documentation - Simplify README, Installation Guide for virtualenv

### PR DESCRIPTION
**Pull request by Rafficer**
The README had a lot of duplicate information with the USAGE.md file. It just contains the most necessary information now and otherwise links to the usage guide.

The Usage guide now includes a guide for installing in a virtualenv. (see #15)

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -121,6 +125,76 @@ Bye Bye 😔
...
+
+    `cd ~/Documents/ProtonVPN-CLI`
+
+2. Install the virtualenv Python package
+
+    `pip3 install virtualenv --user`
+
+3. Create a virtual environment and activate it
+
+    `virtualenv .venv`
```
Comment (kaplun) [inline USAGE.md]: Can't you simply do `virtualenv ~/Documents/ProtonVPN-CLI`

File: USAGE.md [hunk 2]
```diff
@@ -121,6 +125,76 @@ Bye Bye 😔
...
 
+### Installing in a virtual environment
+
+If you're having trouble with the normal installation or don't want to install ProtonVPN-CLI as root, follow this guide to install it in a Python virtual environment.
+
+#### Install
+
+1. Create a directory for the Installation and step into it
+
+    `mkdir ~/Documents/ProtonVPN-CLI`
```
Comment (kaplun) [inline USAGE.md]: Hum.. Maybe better under `~/ProtonVPN-CLI` directly

File: USAGE.md [hunk 3]
```diff
@@ -121,6 +125,76 @@ Bye Bye 😔
...
+
+    `which protonvpn`
+
+6. If that works, deactivate the virtual environment again
+
+    `deactivate`
+
+7. Link the executable from above (output of the `which` command) to a PATH folder so you can access it from anywhere
+
+    `sudo ln -sf ~/Documents/ProtonVPN-CLI/.venv/bin/protonvpn /usr/local/bin/protonvpn`
```
Comment (kaplun) [inline USAGE.md]: Maybe better add the path to the PATH variable?

Comment (Rafficer) [reply USAGE.md]: I don't think that's right. The `.venv/bin/` directory also contains executables for interacting with the virtual environment as well as the pip and python binaries for the virtual environment.

**Comment by Rafficer** [comment]
I've updated everything with the proposed changes.

**Pull request merged**